Server/World.cs

```diff
@@ -288,7 +288,7 @@ public DataEntry(SaveData data, int typeID, string typeName, long pos, int lengt
 					{
 						Console.WriteLine("Error: Type '{0}' was not found. Delete all of those types? (y/n)", typeName);
 
-						if (Console.ReadKey(true).Key == ConsoleKey.Y)
+						if (Insensitive.Equals(Console.ReadLine(), "Y"))

```

Comment:
Same here, you have a check of `Core.Service` right above. Just change it to use `Core.NoConsole` should be enough.